c/driver_manager/adbc_driver_manager.cc

```diff
@@ -15,26 +15,42 @@
 // specific language governing permissions and limitations
 // under the License.
 
-#include "arrow-adbc/adbc_driver_manager.h"
+#if defined(_WIN32)
+#pragma comment(lib, "advapi32.lib")  // for registry stuff
+#include <windows.h>                  // Must come first
+
+#include <libloaderapi.h>
+#include <string.h>  // _wcsnicmp
+#include <strsafe.h>
+
+#include <codecvt>
```

Comment:
This is deprecated in C++17...

Reply:
Since we only need the conversions on Windows, maybe we can use Win32 instead? https://learn.microsoft.com/en-us/previous-versions/bb202786(v=msdn.10)?redirectedfrom=MSDN